### ISSUE TITLE
prqlc CLI not built with cli feature enabled (version 0.11.2 and 0.11.3)

### ISSUE THREAD

**Issue by tryan-dev** [PRQL/prql#4254]
### What happened?

This is my first time attempting to use prql.

Most of the downloads I have tried, for multiple platforms, do not work from the cli. Invoking the executable from a shell prompt results in the following message:

```
PS C:\Users\TimRyan\code\prql\prqlc-v0.11.3-x86_64-pc-windows-msvc> .\prqlc.exe --help
thread 'main' panicked at prqlc\prqlc\src\main.rs:11:5:
Crate is not built with the `cli` feature enabled, or was built for a wasm target.
```

I installed prql with scoop on Windows 11 (23H2), which I use as a package manager for my work Windows OS. The scoop-installed prql failed to work from PowerShell 7, as well as good old fashioned cmd.exe.

Next, I tried downloading the Windows executable directly from Github and got the same results (displayed above). I also tried the binaries for macOS arm64, and for Linux running under WSL with the same results.

The only binary that has worked for me so far is from Homebrew on macOS.

I'd really like to try prql for work, which will require having prql on Windows, unfortunately. Hoping I'm either missing something obvious, or that there's a simple issue with your CI pipeline.

### PRQL input

```elm
prqlc.exe --help
```

### SQL output

```SQL
thread 'main' panicked at prqlc\prqlc\src\main.rs:11:5:
Crate is not built with the `cli` feature enabled, or was built for a wasm target.
```

### MVCE confirmation

- [x] Minimal example
- [X] New issue

**Comment by eitsupi** [comment]
I can reproduce this.
The last version that worked seems to be 0.11.1.
https://github.com/PRQL/prql/releases/tag/0.11.1

You can install with `winget install prqlc --version 0.11.1`

I'm surprised no one has noticed this in a month (i.e. no one is using prqlc!)

**Comment by eitsupi** [comment]
I think this line causes this. `--no-default-features` should be removed.
https://github.com/PRQL/prql/blob/26940ddef107bcd81b89876b788b082fb8974ced/.github/actions/build-prqlc/action.yaml#L71

@max-sixty I think we should create a patch release as sooon as possible.

**Comment by max-sixty** [comment]
Sorry, this was my fault. Thanks for spotting @tryan-dev

**Comment by max-sixty** [comment]
Released 0.11.4

**Comment by tryan-dev** [comment]
I logged in this morning and scoop had the patch release already. Thanks everyone!

Also, thank you for putting this project out into the world. I'm really excited about what you've come up with here. I think this could be a real game-changer. 👍
